src/include/platform/ConnectivityManager.h

```diff
@@ -162,6 +166,8 @@ class ConnectivityManager
     bool IsThreadProvisioned();
     void ErasePersistentInfo();
     bool HaveServiceConnectivityViaThread();
+    CHIP_ERROR GetThreadNetworkDiagnosticAttributeInfo(chip::AttributeId attributeId, uint8_t ** buffer, uint16_t & ReadLength,
```

Comment:
Need to document the semantics of this method, especially around `buffer` and `readLength`.